src/filesystem.hpp

```diff
@@ -428,7 +428,7 @@ std::string get_short_wml_path(const std::string &filename);
  *   units/konrad-fighter.png ->
```

Comment:
Remember to update the comments too. The line that needs changes is above this, on line 425.

Reply:
You mean the word "image"?

Reply:
You mean the word "image"?

Reply:
More than just one word - the documentation needs to explain what the new argument means.

The example in the documentation could do with words too, and needs to be an example of how to use the current version of the function, not just the previous version of the function.